odoo/addons/base/data/ir_cron_data.xml

```diff
@@ -19,4 +19,14 @@
         <field name='interval_type'>days</field>
         <field name="numbercall">-1</field>
     </record>
+
+    <record id="ir_cron_language_update" model="ir.cron">
+        <field name="name">Base: Update installed language translations</field>
+        <field name="model_id" ref="base.model_base_language_install"/>
+        <field name="state">code</field>
+        <field name="code">model.auto_update()</field>
```

Comment:
do we actually need to create a specific method? can't we just the three lines of code here?